content/tutorials/git_authentication/index.Rmd

```diff
@@ -0,0 +1,236 @@
+---
+title: "Git Authentication"
```

Comment:
```suggestion
title: "Git Authentication on Windows"
```

ssh protocol with a password protected private key is IMHO still more secure than a token with lots of permisions.
ssh is quite straightforward to use on Linux. Hence I'd only recommend this on Windows.